framework/src/st_things/things_stack/src/common/security/sss_security/sss_storage_server.c

```diff
@@ -0,0 +1,115 @@
+/****************************************************************************
+ *
+ * Copyright 2017 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http;//www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+///
+/// This sample provides the way to create cloud sample
+#include <stdio.h>
+#include <string.h>
+#include <pthread.h>
+#include <unistd.h>
+#include <stdlib.h>
+#include <signal.h>
+#include <time.h>
+#include "cacommon.h"
+#include "cacommonutil.h"
+#include "ocstack.h"
+#include "ocpayload.h"
+#include "logger.h"
+#include "mbedtls/ssl_ciphersuites.h"
+
+#include "things_malloc.h"
+#include "things_logger.h"
+
+//pkssss #include "mbedtls/sss_storage.h"
+//#include "mbedtls/sss_storage.h"
+
+/** Persistent storage open handler.*/
+FILE *secure_fopen(const char *path, const char *mode)
```

Comment:
It seems that functions 'secure_fopen/fread/fwrite/fclose' are dead code.
The same functions are defined  in st_things/things_stack/src/common/security/securestorage/src/ss_api.c.